tools/docker/build.sh

```diff
@@ -17,9 +17,20 @@ main() {
         run ${scriptdir}/image-build.sh
     }
 
-    run docker container run --workdir=$rootdir --user=${SUDO_UID:-$(id -u)}:${SUDO_GID:-$(id -g)} \
-        -e USER=${SUDO_USER:-${USER}} -v ${rootdir}:${rootdir} \
-        --rm prplmesh-builder $@
+    # Default docker arguments
+    docker_args="\
+    --workdir=$rootdir --user=${SUDO_UID:-$(id -u)}:${SUDO_GID:-$(id -g)} \
+    -e USER=${SUDO_USER:-${USER}} -v ${rootdir}:${rootdir} \
+    "
+
+    # Add platform base directory mapping into the container
+    if [ ! -z "${PRPLMESH_PLATFORM_BASE_DIR}" ]; then
+        docker_args="${docker_args} \
+        -v ${PRPLMESH_PLATFORM_BASE_DIR}:${PRPLMESH_PLATFORM_BASE_DIR} \
+        "
+    fi
+
+    run docker container run ${docker_args} --rm prplmesh-builder $@
```

Comment:
Is this the only change needed to build in docker for a platform? That's nice. Only... why would you want to build in docker for a platform if the platform stuff is not in the docker image itself? That's the part I never understood about Dockerfile.openwrt...

Reply:
It's a temporary solution until we'll be able to package all target platforms into docker containers... Till then, I prefer to use a generic builder docker with a pre-built target tree.